cli/src/cluster_query.rs

```diff
@@ -31,6 +35,18 @@ pub trait ClusterQuerySubCommands {
 impl ClusterQuerySubCommands for App<'_, '_> {
     fn cluster_query_subcommands(self) -> Self {
         self.subcommand(
+            SubCommand::with_name("catchup")
+                .about("Wait for a validator to catch up to the cluster")
+                .arg(
+                    Arg::with_name("node_pubkey")
+                        .index(1)
+                        .takes_value(true)
+                        .value_name("PUBKEY")
+                        .validator(is_pubkey_or_keypair)
```

Comment:
This arg should be `required`, instead of unwrapping in `parse_catchup()`. However, see comment below.